README.md

```diff
@@ -30,22 +30,22 @@ This module supports Puppet Enterprise and Puppet versions 3.8 or newer. For old
 1. On each Puppet agent, make sure the [`pluginsync`](https://docs.puppet.com/puppet/latest/configuration.html#pluginsync) and [`report`](https://docs.puppet.com/puppet/latest/configuration.html#report) settings are enabled. (These settings are normally enabled by default.)
 
   ```
-[agent]
+[main]
```

Comment:
Was there a reason this was changed from agent -> main?

Reply:
Just mirroring what I found in the documentation while double-checking what what master should be changed to. See here: https://puppet.com/docs/puppet/7.0/config_file_main.html